index.html

```diff
@@ -173,13 +173,16 @@ <h2>Introduction</h2>
       </p>
       <ol>
         <li>
-what a verifiable credential contains,
+what a verifiable credential is and how it's comprised,
         </li>
         <li>
-an ecosystem where verifiable credentials are expected to be useful, and
+what a presentation presentation (derived from verifiable credential) is and how it's comprised,
```

Comment:
`presentation presentation` => `verifiable presentation`

I don't think that the parenthetical `(derived from verifiable credential)` is useful here -- a link to terminology would suffice. There was also a typo in it, but we can just remove it, IMO.

Reply:
Fixed, thanks.